.ebextensions/rails.config

```diff
@@ -11,6 +11,13 @@ files:
 
       cd $(/opt/elasticbeanstalk/bin/get-config container -k app_deploy_dir)
 
-      su -s /bin/bash -c 'DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bin/rails db:setup' $(/opt/elasticbeanstalk/bin/get-config container -k app_user)
+      if [ "$EB_ENVIRONMENT" == "move-mil-prod" ]
+      then
+        # In production, ensure that DB migrations run because this might be a new version of the app that requires it, but do not blow the DB away and re-seed
+        su -s /bin/bash -c 'DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bin/rails db:migrate' $(/opt/elasticbeanstalk/bin/get-config container -k app_user)
```

Comment:
Database migrations are handled in a "pre" hook during deploy. Check out `/opt/elasticbeanstalk/hooks/appdeploy/pre/12_db_migration.sh` on one of the servers. There might be some useful logic there that could be used here.

Reply:
In other words, I don't need to worry about it?

Reply:
Correct. You shouldn't have to run migrations by hand in this script.